frontend/apps/filemanager/filemanagerbookinfo.lua

```diff
@@ -400,17 +402,26 @@ function BookInfo:setCustomMetadata(file, book_props, prop_key, prop_value)
         os.remove(custom_doc_settings.custom_metadata_file)
         DocSettings:removeSidecarDir(file, util.splitFilePathName(custom_doc_settings.custom_metadata_file))
     else
+        if book_props.pages then
+            local original_props = custom_doc_settings:readSetting("doc_props")
```

Comment:
Might need a comment. I asume this `if book_props.pages` is to know if it comes from the SQL bookinfo? Or else?
Also, why is it needed to save an updated doc_props.pages in the custom_metadata.lua ? Will it be updated if we later change font size - or not until we again change custom metadata?

Reply:
Why do we need to save it?
It feels it shouldn't need to be in there, as it's not really metadata, and is not something that relates to what we have in custom_props.
```lua
return {
    ["custom_props"] = {
        ["title"] = "cjkpunct44",
    },
    ["doc_props"] = {
        ["pages"] = 2,
        ["title"] = "cjkpunct",
    },
}
```
(If not saved, no question to ask about when to update it :)

Reply:
It can be used in
https://github.com/koreader/koreader/blob/86420ca998d5beeb7cef13c1c7a48d72c47f2d2f/frontend/apps/filemanager/filemanagerbookinfo.lua#L224-L231

Reply:
We can open a book, then reset `doc_settings` but keep custom metadata.
And have number of pages.

Reply:
Well, ok if you feel it's valuable.
But in other situations (when we have a docsettings present), we are sure an updated `pages` will be used, and not this possibly obsolete/innacurate one?